### PR TITLE
Updated npm scripts

### LINKED ISSUE #1939: Audit of npm scripts
**Describe the bug**
Continuation of https://github.com/bbc/simorgh/issues/1418 in psammead for consistency.

**Describe the solution you'd like**
Simplify the scripts and naming them better so that they are self-explanatory so that we do not need to maintain documentation for scripts

**Testing notes**
All the npm scripts available in `package.json` should run fine on CI and locally.

Dev insight: This will be documentation so no manual testing. However, a tester should be able to read through the documentation and understand each

### PR DESCRIPTION
**Pull request by hotinglok**
Resolves #1939

**Overall change:**
Reduced overall number of scripts from 29 to 21 through removing redundant scripts and merging scripts together.

**Code changes:**
Replaced/Merged into to other scripts
- ```test:lint:css``` merged into ```test:lint```
- ```forced-package-lock-https``` made into script in /scripts
- ```test:chromatic``` moved into MakeFile

Removals
- ```updateSnapshots```  
Use ```npm run test:unit -- -u``` instead.
- ```test:unit:watch``` 
Use ```npm run test:unit --watch``` instead.
- ```test:packages``` 
Use ```npx lerna run test``` instead.
- ```install:packages:link```
Use ```npm run install:packages --force-local``` instead.

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] This PR requires manual testing

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,30 +9,23 @@
...
     "storybook": "NODE_ENV=development start-storybook -p 8180 -c .storybook",
     "storybook:ie11": "export FIX_STORYBOOK_IE11=true && npm run storybook",
-    "test": "npm run test:lint && npm run test:lint:css && npm run test:unit && npm run test:packages",
-    "test:chromatic": "chromatic test run --storybook-build-dir=storybook_dist || true",
-    "test:packages": "lerna run test",
-    "test:lint": "eslint --ext .js,jsx,json ./packages ./scripts",
-    "test:lint:css": "stylelint 'packages/**/*.js' 'packages/**/*.jsx' 'scripts/**/*.js'",
-    "test:unit": "npm run build && jest --verbose --coverage",
-    "test:unit:watch": "npm run build && jest --verbose --coverage --watch",
     "talos": "node scripts/talos/cli",
```
Comment (dr3) [inline package.json]: We can remove talos from here, and just call it from the make file, it shouldnt really we called locally :P

**Review by dr3 — CHANGES_REQUESTED** [review]
LGTM just one tweak, can you make sure all readmes are updated too, if needed :)

File: package.json [hunk 2]
```diff
@@ -9,31 +9,24 @@
     "build:cjs": "lerna exec --parallel -- babel src -d dist --ignore $npm_package_config_babel_ignore --root-mode upward",
     "build:esm": "BABEL_TYPE=esm lerna exec --parallel -- babel src -d esm --configFile babel.config.js --ignore $npm_package_config_babel_ignore --root-mode upward",
     "build:storybook": "build-storybook -o storybook_dist",
-    "ci:packages": "npm ci && lerna exec -- npm ci",
-    "changeScanner": "node scripts/changeScanner",
-    "create:package": "yo .yeoman",
     "bumpPackages": "node scripts/bumpPackages/cli",
     "bumpChangelogs": "node scripts/bumpChangelogs/cli",
-    "regeneratePackageLocks": "node scripts/regeneratePackageLocks",
+    "bumpPackages": "node scripts/bumpPackages/cli",
```
Comment (dr3) [inline package.json:13]: This is now in here twice? line 12 is a duplicate thats out of alphabetical order

File: package.json [hunk 3]
```diff
@@ -9,31 +9,24 @@
     "build:cjs": "lerna exec --parallel -- babel src -d dist --ignore $npm_package_config_babel_ignore --root-mode upward",
     "build:esm": "BABEL_TYPE=esm lerna exec --parallel -- babel src -d esm --configFile babel.config.js --ignore $npm_package_config_babel_ignore --root-mode upward",
     "build:storybook": "build-storybook -o storybook_dist",
-    "ci:packages": "npm ci && lerna exec -- npm ci",
-    "changeScanner": "node scripts/changeScanner",
-    "create:package": "yo .yeoman",
     "bumpPackages": "node scripts/bumpPackages/cli",
```
Comment (dr3) [inline package.json]:
```suggestion
```

**Review by dr3 — APPROVED** [review]
Great work :)

**Review by HarveyPeachey — APPROVED** [review]
LGTM, do you need to update the changelog.md with these changes?

**Review by chris-hinds — APPROVED** [review]

**Pull request merged**